app/js/lib/foxbox/settings.js

```diff
@@ -62,6 +62,8 @@ export default class Settings extends Model {
     return new Promise(resolve => {
       // @todo Remove only the items set here.
       storage.clear();
```

Comment:
__note:__ also not related to your patch, but it's funny that fake storage defined above doesn't have ```clear``` method, so with fake storage this promise will never resolve :)